src/main/amp/config/alfresco/module/com.arondor.arender.alfresco.content.transformer/alfresco-global.properties

```diff
@@ -171,3 +171,4 @@ content.transformer.arender2pdf.extensions.xltx.pdf.priority=30
 
 content.transformer.arender2pdf.extensions.zip.pdf.supported=true
 content.transformer.arender2pdf.extensions.zip.pdf.priority=30
+content.transformer.arender2pdf.url=http://10.233.18.244:8761/
```

Comment:
Do not set a URL here. It must be provided by the user from `alfresco-global.properties`, and no default should be set here